metacat-common-server/src/main/java/com/netflix/metacat/common/server/spi/MetacatCatalogConfig.java

```diff
@@ -49,10 +53,11 @@ private MetacatCatalogConfig(
      * Creates the config.
      *
      * @param type       type
+     * @param catalogName catalog name
      * @param properties properties
      * @return config
      */
-    public static MetacatCatalogConfig createFromMapAndRemoveProperties(final String type,
+    public static MetacatCatalogConfig createFromMapAndRemoveProperties(final String type, final String catalogName,
```

Comment:
You should probably either put all of these on new lines or keep them all in one line. Otherwise it's easy to miss seeing parameters.